.github/workflows/build-macos.yml

```diff
@@ -59,59 +59,26 @@ jobs:
         id: paths
         run : |
           echo "build_dir=${{ github.workspace }}/build" >> $GITHUB_OUTPUT
-          echo "ext_deps_dir=${{ github.workspace }}/external_dependencies" >> $GITHUB_OUTPUT
           echo "install_dir=${{ github.workspace }}/install" >> $GITHUB_OUTPUT
 
-      # Step: Install system-provided dependencies
```

Comment:
Please restore these steps and disable them for macOS builds. They are needed for ubuntu builds and potentially windows (as much as I hate saying this) in the future.